### DISCUSSION TITLE
Textarea auto height

### DISCUSSION THREAD

**Discussion by xmrhahnx** [Textualize/textual#4320] [Q&A]
Hi there,
I placed a disabled TextArea inside a Collapsible, just to show some read-only text but the TextArea is a lot heigher than its content. Is there any way to setup the TextArea to have the same height of its content?

Thank you

**Comment by davep** [comment]
You appear to to have had the answer in your question's subject: you should be able to set its [`height`](https://textual.textualize.io/styles/height/) to [`auto`](https://textual.textualize.io/css_types/scalar/#auto).
